e2e-tests/golden_path.spec.ts

```diff
@@ -0,0 +1,108 @@
+import { test, expect, Page } from '@playwright/test';
+
+const APP_URL = process.env.TEST_E2E_APP_URL  || 'http://localhost:3000/';
+const USERNAME = process.env.TEST_E2E_USERNAME || '';
+const USER_PASSWORD = process.env.TEST_E2E_PASSWORD || '';
+
+test.describe.serial('Golden path', () => {
+  test.skip(() => {
+    if(!USERNAME || !USER_PASSWORD)
+      console.log('Please provide credentials!');
+    return !USERNAME || !USER_PASSWORD
+  }, 'Please provide credentials!');
+
+  let page: Page;
+
+  test.beforeAll(async ({ browser }) => {
+    page = await browser.newPage();
+  });
+
+  test.afterAll(async () => {
+    await page.close();
+  });
+
+  const workflowRandomName = `qri_test_${Math.floor(new Date().valueOf() / 1000)}`;
+
+  test('Log in', async () => {
+    await page.goto(APP_URL);
+    await page.click('#user_menu_login_button');
+    await page.fill('#username', USERNAME);
+    await page.fill('#password', USER_PASSWORD);
+    const [response] = await Promise.all([
+      page.waitForResponse(resp => resp.url().includes('/oauth/token?grant_type=password')),
+      await page.click('#login_modal_login_button')
+    ]);
+    await expect(response.status()).toBe(200);
+  });
+
+  test('go to new dataset page', async () => {
+    await page.click('#new_dataset_button');
+    await page.click('#splash_modal_workflow_button');
+    await expect(page.url()).toBe(APP_URL+'workflow/new');
+  });
+
+  test('Dry run', async () => {
+    await page.click('#setup-cell .monaco-editor');
+    await page.keyboard.type('\nprint("test")');
+    const selected = await page.$$('.run_bar_run_button');
+    const [response] = await Promise.all([
+      page.waitForResponse(resp => resp.url().includes('/auto/apply')),
+      selected[1].click()
+    ]);
+    await expect(response.status()).toBe(200);
```

Comment:
let's add a few tests here ensuring that the basic expectations of running a dataset have been met:
- our print of "test" is shown
- the resulting dataset is shown in the "Results" section
- In the sidebar: setup, download, and transform show "success" check marks
- In the sidebar: the result section is not empty
- editing the transform should "clear" all the above ui